content/better-bibtex.ts

```diff
@@ -866,23 +866,15 @@ export class BetterBibTeX {
         // onRender: ({ body, item, editable, tabType }) => {
         onRender: ({ body, item, setSectionSummary }) => {
           const citekey = item.getField('citationKey')
-          const textbox = body.ownerDocument.getElementById('better-bibtex-citation-key')
```

Comment:
This will always return the first matched element, which is the one in the library. Thus, the citation key in readers will never be rendered.